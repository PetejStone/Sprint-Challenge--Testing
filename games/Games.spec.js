const db = require('../data/dbConfig.js');
const {insert, remove} = require('./gamesModel.js')
const supertest = require('supertest');
const server = require('../api/server.js');
require('jest-dom/extend-expect')

describe('GET of game', () => {
    //clear data base before each test
    // beforeEach(async ()=> {
    //     await db.truncate()
    // })

    it('should return status 200', () => {
        return supertest(server)
        .get('/games')
        .expect(200)
    })

    // //testing actual content that is returned (empty array for now)
    it('should return []', () => {
        let emptyArray = []
        supertest(server)
        .get('/games')
        expect([]).toEqual(expect.arrayContaining(emptyArray))
        
    })

})

describe('Inserting game', () => {
    // beforeEach(async ()=> {
    //     await db.truncate()
    // })

    
})