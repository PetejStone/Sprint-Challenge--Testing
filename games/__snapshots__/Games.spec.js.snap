// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inserting game should add game 1`] = `"received is not a function"`;
